codegen-server-test/build.gradle.kts

```diff
@@ -25,7 +25,8 @@ data class CodegenTest(val service: String, val module: String, val extraConfig:
 val CodegenTests = listOf(
     CodegenTest("com.amazonaws.simple#SimpleService", "simple"),
     CodegenTest("aws.protocoltests.restjson#RestJson", "rest_json"),
-    CodegenTest("com.amazonaws.ebs#Ebs", "ebs")
+    CodegenTest("com.amazonaws.ebs#Ebs", "ebs"),
```

Comment:
probably not in scope for this PR, but we should add the restXml protocol tests here:https://github.com/awslabs/smithy-rs/blob/cf46613360c184c5a45b7299858ebc72b9b71013/codegen-test/build.gradle.kts#L35-L38

but I suspect it might be a bit rough until we iron out all of the issues 😅

Reply:
It in plans. We will probably work on this next week.